### PR TITLE
[FLINK-25370][docs] update HBase SQL Connector Options table-name des…

### PR DESCRIPTION
**Pull request by lipusheng**
…cription

## What is the purpose of the change

In document the HBase SQL Connector Options "table-name" description is "The name of HBase table to connect.". All demo of HBase SQL Connector document 'table-name' just give the 'tablename', it does't explain how to use the table in the specified namespace of HBase. So I think we need to add some description for this options. Update HBase SQL Connector Options "table-name" description.

## Brief change log

*(for example:)*
  - *The TaskInfo is stored in the blob store on job creation time as a persistent artifact*
  - *Deployments RPC transmits only the blob storage reference*
  - *TaskManagers retrieve the TaskInfo from the blob cache*

## Verifying this change

Please make sure both new and modified tests in this PR follows the conventions defined in our code quality guide: https://flink.apache.org/contributing/code-style-and-quality-common.html#testing

*(Please pick either of the following options)*

This change is a trivial rework / code cleanup without any test coverage.

*(or)*

This change is already covered by existing tests, such as *(please describe tests)*.

*(or)*

This change added tests and can be verified as follows:

*(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (100MB)*
  - *Extended integration test for recovery after master (JobManager) failure*
  - *Added test that validates that TaskInfo is transferred only once across recoveries*
  - *Manually verified the change by running a 4 node cluser with 2 JobManagers and 4 TaskManagers, a stateful streaming program, and killing one JobManager and two TaskManagers during the execution, verifying that recovery happens correctly.*

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): (yes / no)
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: (yes / no)
  - The serializers: (yes / no / don't know)
  - The runtime per-record code paths (performance sensitive): (yes / no / don't know)
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Kubernetes/Yarn, ZooKeeper: (yes / no / don't know)
  - The S3 file system connector: (yes / no / don't know)

## Documentation

  - Does this pull request introduce a new feature? (yes / no)
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)

### CODE REVIEW

**Comment by MartijnVisser** [comment]
@AHeise Please merge

**Comment by lipusheng** [comment]
@AHeise Could you help merge it

File: docs/content/docs/connectors/table/hbase.md [hunk 1]
```diff
@@ -105,7 +105,7 @@ Connector Options
       <td>required</td>
       <td style="word-wrap: break-word;">(none)</td>
       <td>String</td>
-      <td>The name of HBase table to connect.</td>
+      <td>The name of HBase table to connect. Default the table in 'default' namespace, HBase table in specified namespace need to use 'namespace:table'.</td>
```
Comment (JingGe) [inline docs/content/docs/connectors/table/hbase.md:108]:
```suggestion
      <td>The name of HBase table to connect. By default, the table is in 'default' namespace. To assign the table a specified namespace you need to use 'namespace:table'.</td>
```

**Review by JingGe — CHANGES_REQUESTED** [review]

**Comment by JingGe** [comment]
Thanks for your contribution. LGTM.

**Comment by MartijnVisser** [comment]
@flinkbot run azure

**Comment by JingGe** [comment]
@fapaul please help merging, thanks.

**Review by fapaul — APPROVED** [review]
LGTM @lipusheng can you squash your two commits to one?

**Comment by lipusheng** [comment]
@fapaul Yes, I have squash to commits to one. Please help with reviews, thanks.
@JingGe I have squash you commit to my code. Thanks for you reviews and commit. Please help with reviews again, thanks.

**Review by JingGe — APPROVED** [review]
LGTM

**Comment by lipusheng** [comment]
@fapaul can you help me merge it, thanks.

**Comment by lipusheng** [comment]
@MartijnVisser can you help me merge this pull request, thanks.

**Comment by MartijnVisser** [comment]
@lipusheng It's currently holiday week in Western Europe, so merged are a bit slower than normal since a lot of people are on a break. It will get merged soonish.

**Comment by lipusheng** [comment]
Ok, thanks.

**Review by fapaul — CHANGES_REQUESTED** [review]
I had another look and we should also update https://github.com/apache/flink/blob/1166d11f61ae10b3f3ae71825360cbf6c21eda4d/flink-connectors/flink-connector-hbase-base/src/main/java/org/apache/flink/connector/hbase/table/HBaseConnectorOptions.java#L37 

Usually, we first update the config options and then generate the documentation via something like 
```
mvn package -Dgenerate-config-docs -pl flink-docs -am -nsu -DskipTests
```

**Comment by JingGe** [comment]
@fapaul 
> Usually, we first update the config options and then generate the documentation via something like
> 
> ```
> mvn package -Dgenerate-config-docs -pl flink-docs -am -nsu -DskipTests
> ```

good catch! Thanks for the hint! The content now is not dynamically created from the Options. I will created a follow-up ticket for it. There is one more common question which is good for contributors to know: How could the Chinese content be generated with the mvn command since there is only content in English defined in the Options classes.

**Comment by fapaul** [comment]
> good catch! Thanks for the hint! The content now is not dynamically created from the Options. I will created a follow-up ticket for it. There is one more common question which is good for contributors to know: How could the Chinese content be generated with the mvn command since there is only content in English defined in the Options classes?

AFAIK the content is automatically updated if you run the command. So it may not need a follow-up. Indeed we have a slight problem with our Chinese doc because they are not generated. I do not have a good answer for this right now.

**Review by fapaul — APPROVED** [review]
Thanks for the quick update

**Pull request merged**